extensions/replication/ReplicationQueuePopulator.js

```diff
@@ -52,7 +52,24 @@ class ReplicationQueuePopulator extends QueuePopulatorExtension {
                      `${queueEntry.getBucket()}/${queueEntry.getObjectKey()}`,
                      JSON.stringify(entry));
 
-        this._incrementMetrics(entry.bucket, queueEntry.getBytesMetric());
```

Comment:
It seems we don't provide per-bucket metrics anymore, is it expected?

Reply:
Previously, I had collected metrics by bucket, but was never used. Because of time constraints, I decided to go ahead and swap that out for `site`. 

Also, important to note is we want to keep redis keys as low as possible. Each new layer adds exponentially more keys. Adding sites parameters increased keys by the number of sites :/